works.xml

```diff
@@ -139550,6 +139550,12 @@ Mittelalterliche und frühneuzeitliche deutsche Übersetzungen  -->
                <textLang mainLang="la"/>
                <term ref="#subject_theology"/>
             </bibl>
+            <bibl xml:id="work_17521">
+               <author key="person_3288237">Otto, von Passau, active 1386</author>
+               <title type="uniform">Otto, von Passau, active 1386: Die vierundzwanzigsten Alten</title>
+               <textLang mainLang="gmh"/>
+               <term ref="#subject_theology"/>
```

Comment:
please add bibliography - just a link to Handschriftencensus record would probably be sufficient